mlflow-v2-examples/kserve-example.ipynb

```diff
@@ -0,0 +1,440 @@
+      "Requirement already satisfied: pyparsing>=2.3.1 in /home/pocik/.pyenv/versions/3.8.0/envs/pip-compile8/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.1.1) (3.0.9)\n",
+      "Requirement already satisfied: cycler>=0.10 in /home/pocik/.pyenv/versions/3.8.0/envs/pip-compile8/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.1.1) (0.11.0)\n",
+      "Requirement already satisfied: pillow>=6.2.0 in /home/pocik/.pyenv/versions/3.8.0/envs/pip-compile8/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.1.1) (9.5.0)\n",
+      "Requirement already satisfied: fonttools>=4.22.0 in /home/pocik/.pyenv/versions/3.8.0/envs/pip-compile8/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.1.1) (4.39.4)\n",
+      "Requirement already satisfied: charset-normalizer<4,>=2 in /home/pocik/.pyenv/versions/3.8.0/envs/pip-compile8/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.1.1) (3.1.0)\n",
+      "Requirement already satisfied: idna<4,>=2.5 in /home/pocik/.pyenv/versions/3.8.0/envs/pip-compile8/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.1.1) (3.4)\n",
+      "Requirement already satisfied: certifi>=2017.4.17 in /home/pocik/.pyenv/versions/3.8.0/envs/pip-compile8/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.1.1) (2023.5.7)\n",
+      "Requirement already satisfied: tqdm>4.25.0 in /home/pocik/.pyenv/versions/3.8.0/envs/pip-compile8/lib/python3.8/site-packages (from shap<1,>=0.40->mlflow==2.1.1) (4.64.1)\n",
+      "Requirement already satisfied: numba in /home/pocik/.pyenv/versions/3.8.0/envs/pip-compile8/lib/python3.8/site-packages (from shap<1,>=0.40->mlflow==2.1.1) (0.56.4)\n",
+      "Requirement already satisfied: slicer==0.0.7 in /home/pocik/.pyenv/versions/3.8.0/envs/pip-compile8/lib/python3.8/site-packages (from shap<1,>=0.40->mlflow==2.1.1) (0.0.7)\n",
+      "Requirement already satisfied: greenlet!=0.4.17 in /home/pocik/.pyenv/versions/3.8.0/envs/pip-compile8/lib/python3.8/site-packages (from sqlalchemy<2,>=1.4.0->mlflow==2.1.1) (2.0.2)\n",
+      "Requirement already satisfied: smmap<6,>=3.0.1 in /home/pocik/.pyenv/versions/3.8.0/envs/pip-compile8/lib/python3.8/site-packages (from gitdb<5,>=4.0.1->gitpython<4,>=2.1.0->mlflow==2.1.1) (5.0.0)\n",
+      "Requirement already satisfied: llvmlite<0.40,>=0.39.0dev0 in /home/pocik/.pyenv/versions/3.8.0/envs/pip-compile8/lib/python3.8/site-packages (from numba->shap<1,>=0.40->mlflow==2.1.1) (0.39.1)\n",
+      "\n",
+      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m A new release of pip available: \u001b[0m\u001b[31;49m22.3.1\u001b[0m\u001b[39;49m -> \u001b[0m\u001b[32;49m23.3\u001b[0m\n",
+      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m To update, run: \u001b[0m\u001b[32;49mpip install --upgrade pip\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "# first install necessary libs\n",
```

Comment:
I think this should be in the form of heading/section outside the cell to improve readability.